src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -5,20 +5,28 @@
     display: flex;
     align-items: center;
     justify-content: center;
+    padding: 0 6px;
 }
 
 .bp-ColorPickerControl-toggle {
     @include bp-Control;
 
+    padding: 4px 0 4px 5px;
+
     &.bp-is-active,
     &:focus,
     &:hover {
-        .bp-ColorPickerControl-toggle-background {
+        .bp-ColorPickerControl-toggle {
             padding: 8px;
             background-color: $fours;
             border-radius: $bp-controls-radius-inner;
         }
     }
+
+    svg {
```

Comment:
Why do we have to explicitly set the width and height of the svg element?

Reply:
The width and height in the controlbar are smaller than the default `1em` svg size.